src/search/heuristic.cc

```diff
@@ -66,6 +66,16 @@ void Heuristic::add_options_to_feature(plugins::Feature &feature) {
     add_evaluator_options_to_feature(feature);
 }
 
+
+tuple<shared_ptr<AbstractTask>, bool, string, utils::Verbosity> Heuristic::get_heuristic_parameters_from_options(const plugins::Options &opts){
+    auto parent_parameter_tuple = get_evaluator_parameters_from_options(opts);
+    auto own_parameter_tuple = make_tuple(
+        opts.get<shared_ptr<AbstractTask>>("transform"),
+        opts.get<bool>("cache_estimates")
+    );
+    return tuple_cat(own_parameter_tuple, parent_parameter_tuple);
+}
```

Comment:
If we use this kind of pattern more frequently, it may be worth creating our own utility method that creates a tuple from elements and a "base" tuple in one go, along the lines of:

```
return make_flat_tuple(
    opts.get<shared_ptr<AbstractTask>>("transform"),
    opts.get<bool>("cache_estimates"),
    get_evaluator_parameters_from_options(opts));
```

If we only use this in cases where we have a feature class that is the same time a derived class and a base class (like Heuristic), it might not be worth it though, because we prefer flat inheritance hierarchies anyway. For example, the Evaluator/Heuristic distinction is something we have long wanted to get rid of, but didn't prioritize so far. (We did get rid of yet another intermediate class a long time ago though, and we reduced the differences between the two classes.)